### ISSUE TITLE
Incorrect (or confusing) balance history.

### ISSUE THREAD

**Issue by musicslayer** [solana-labs/solana#20777]
#### Problem
I'm examining the account AqR6BBUFG3NhC4bSAU31zhHNZ1JdSzPwM43PKSSG878N in Solscan:

https://solscan.io/account/AqR6BBUFG3NhC4bSAU31zhHNZ1JdSzPwM43PKSSG878N

and looking and some of the transactions, and in particular the pre and post balances. I see two discrepancies:

1) The very last transaction to take place was this one:

https://solscan.io/tx/52VTG4rL5hstFqvwMcm5y5F9x959Xou5s4V23f6y6UT6fcy7HD3yLLXLTBCo5kxiFF9wvXFayBkMupLSsgty4fvM

And at the end, this account has a balance of 0.00035.

Yet... the current balance is 0. So... where did the 0.00035 go? And if it went somewhere, is there a transaction representing that transfer?

2) The following two transactions are listed one right after the other:
https://solscan.io/tx/5vkAZQpBqfJ677AGtzNY9zZHk2e8rWdcV2pq6efzavqUsymCPckMjoWk8rJFRvDwNevfLAQWHypPeo4du1DjK2zE

https://solscan.io/tx/4VGonPMGKmcvngX9RakPgXkpD3YB2YDQ7bohwnSMLEKM4S7G3g3R9Hk8ogNQQHufLD1YP5iZk4hkHsjq7QibV7FW

At the end of the first one, the balance was 0.00031092, but at the beginning of the second transaction, the balance was 0.000303603.

In other words, between these two transactions 0.000007317 was lost. Where did this amount go? Is this tracked anywhere,

Aside from using Solscan, I manually explored all these transactions using the Solana JSON RPC API, and the ending balances seem to be reported correctly in the explorer. But I still cannot find where these balance discrepancies come from.

**Comment by mvines** [comment]
I responded in Discord.  tldr: rent!
